test/e2e/go.mod

```diff
@@ -166,15 +169,15 @@ require (
 	k8s.io/cli-runtime v0.27.2 // indirect
```

Comment:
I didn't bump the indirect dependencies( line 167-171, 173, and 176 ) to 1.28 related.
They seem indirectly used by cluster api and helm.

Reply:
It's ok

Reply:
Yup should be fine